horizontalintegration/xmc-hztl-foundation-2024-q1
# Bug FIS-392 Footer Layout Issues

## What does this PR do and why?

This PR resolves some issues with how the various footer components are laid out at the various breakpoints.

## Screenshots or screen recordings

| Before | After |
| ------ | ----- |
| ![localhost_3000_ (1)](https://github.com/user-attachments/assets/9802db6c-43bc-44a2-abfd-0ac8c379ec34) | ![localhost_3000_](https://github.com/user-attachments/assets/a2a2b37d-7fa5-4761-a5b6-a7f2eac144fa) |

## How to set up and validate locally

1. Check out the branch _bug/FIS-392-footer-breakpoints_.
2. From a command prompt, navigate to _./src/hztl-foundation_ and execute `npm run start:connected`.
3. From a browser, navigate to http://localhost:3000. Observe the footer components across the breakpoints and confirm that it is rendered correctly.

## PR acceptance checklist

This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
